src/presenters/admin/indexation-list-item-presenter.php

```diff
@@ -59,15 +72,27 @@ public function present() {
 			$output .= '<span class="wpseo-checkmark-ok-icon"></span>' . \esc_html__( 'Great, your site has been optimized!', 'wordpress-seo' );
 		}
 		else {
+
+			$should_index = $this->indexable_helper->should_index_indexables();
+			$disabled     = ( $should_index ) ? '' : 'disabled';
```

Comment:
```suggestion
			$disabled     = ( $should_index ) ? '' : ' disabled';
```

Reply:
unfortunately, this breaks Travis depending on the platform; I've tried several variations and unfortunately this is the only one that succeeds.